runtime/syntax/make.vim

```diff
@@ -2,8 +2,8 @@
 " Language:	Makefile
 " Maintainer:	Roland Hieber <rohieb+vim-iR0jGdkV@rohieb.name>
```

Comment:
How about adding your GitHub account here so that one can easily contact you on GitHub?

Reply:
Good idea!